host/README.md

```diff
@@ -67,12 +68,12 @@ $ security add-trusted-cert -d -k ~/Library/Keychains/login.keychain domain.crt
 
 ### Start the Runner
 
-Start tmux and launch a new runner!
+Install the service and launch it.
 
 ```
-$ tmux
-$ cd vm
-$ ./launch.sh
+$ sudo chown root:wheel launch.sh
+$ sudo cp com.mirego.ekiden /Library/LaunchDaemons
```

Comment:
C'est pas `com.mirego.ekiden.plist` qu'il faut copier? 🤔

Reply:
Yup, forgot to update it there 🙂